tests/integration/conftest.py

```diff
@@ -105,3 +105,10 @@ def pytest_addoption(parser: Parser):
         "to render the bundle definition template."
         "If set to False, the integration tests will be run against charms in Charmhub.",
     )
+    parser.addoption(
+        "--charmcraft-clean",
+        action="store_true",
+        default=False,
```

Comment:
Q: do we need to use `default=False` since we are using `store_true`? from my understanding `store_true` will also set default to `False`

Reply:
you 're right, I missed that detail, updated.